### ISSUE TITLE
Share the route total loop time on the map

### ISSUE THREAD

**Issue by jeremywight** [codeforgreenville/trolley-tracker-api-dot-net#52]
A user came in tonight and shared an idea which I think would be a great addition. She is new to town and found the app and was trying to plan a trip using the app. Since there isn't a schedule she wondered how she would plan ahead. I mentioned that the trolley loop typically takes somewhere between 45-60min, but I realize that is not something you would know. 

She mentioned it would be really helpful to give some idea of how long the full loop takes so that you can plan accordingly. I was thinking it might be good to time the full loop length and then put that on the route when you select it. I realize that each loop will be different, so maybe we average the last 3 loops, but generally, I think that could be a great thing to share, or at least just make a general hardcoded label the loops typically take around an hour. Thoughts @bikeoid @austin9350 @ajhodges ?

**Comment by bikeoid** [comment]
I agree - a good idea.  The tricky parts in automatically measuring loop times are 
 1. Detecting and removing break times
  2. Detecting a detour condition and how to record and show that

  Average time is more like 30-45 minutes, with nearly an hour when the driver takes a break.
